floodlight/io/catapultApi.py

```diff
@@ -0,0 +1,495 @@
+import requests
+import numpy as np
+from floodlight.core.xy import XY
+from typing import List, Dict, Tuple, Union, Any
+from pathlib import Path
+import warnings
+import json
+import os
+from collections import Counter

```

Comment:
Hello @draabe,

i changed now the write_data and the read_data to dump_list_of_dicts_to_pickle and load_list_of_dicts_from_pickle (ill change it to be able to save json or pkl soon) , since the data previously saved did not contain the positional data but just the player information. Hence my question what exactly you mean by "load catataoult data", should we be able to load both singular player_info json, and the data_dict_list  (also from which format do we accept the data to be loaded json pkl  or more) ? or is the data_dict_list enough, which contains the positional data?

The rest is in the making and should be completed soon. 

Thanks!